game-core/src/main/java/games/strategy/engine/pbem/PbemMessagePoster.java

```diff
@@ -93,23 +81,22 @@ public String getSaveGameRef() {
    * @param historyWriter the history writer (which has no effect since save game has already be generated) // todo (kg)
    * @return true if all posts were successful
    */
-  public boolean post(final IDelegateHistoryWriter historyWriter, final String title, final boolean includeSaveGame) {
-    boolean forumSuccess = true;
+  public boolean post(final IDelegateHistoryWriter historyWriter, final String title) {
+    final Optional<IForumPoster> forumPoster = newForumPoster();
+
+    Future<String> forumSuccess = null;
     final StringBuilder saveGameSb = new StringBuilder().append("triplea_");
-    if (forumPoster != null) {
-      saveGameSb.append(forumPoster.getTopicId()).append("_");
+    if (forumPoster.isPresent()) {
```

Comment:
I'm not entirely happy with the change you proposed for the IForumPoster Optional.
Ignoring the fact, that this proposal changes behaviour slightly in case an exception is thrown when starting the event, the whole null usage should probably avoided entirely by using a rejected future instead. I tried messing around with that, but the code just got messier.
I'll try looking at this again on a new day.